Go/Go.sublime-syntax

```diff
@@ -590,6 +590,55 @@ contexts:
     - match: "%"
       scope: invalid.illegal.placeholder.go
 
+  string-template:
+    - match: "{{"
+      scope: punctuation.section.embedded.begin.template.go
+      push:
+        - meta_scope: template.go
+        - match: "\\s-"
+          scope: keyword.operator.right.trim.template.go
+        - match: "-\\s"
+          scope: keyword.operator.left.trim.template.go
```

Comment:
Similar to the right operator, I think `keyword.operator.template.time.left.go` makes more sense.